src/lua/lua_manager.cpp

```diff
@@ -89,17 +86,27 @@ namespace big
 		});
 	}
 
-	void lua_manager::load_module(const std::string& module_name)
+	void lua_manager::load_module(const std::filesystem::path& module_path)
 	{
 		std::lock_guard guard(m_module_lock);
 
+		const auto module_name = module_path.filename().string();
+
 		const auto id = rage::joaat(module_name);
 
 		for (const auto& module : m_modules)
+		{
 			if (module->module_id() == id)
+			{
+				LOG(WARNING) << "Module with the name " << module_name << " already loaded.";
 				return;
+			}
+		}
+
```

Comment:
unneeded space?

Reply:
no? I prefer the code with empty lines between different parts of a function, because it's more spacious/open and pleasant to read.

Reply:
Why would you make a fucking code review over spaces? Don't be pedantic.